src/python/CRABClient/Commands/setdatasetstatus.py

```diff
@@ -0,0 +1,167 @@
+# pylint: disable=consider-using-f-string, unspecified-encoding
+"""
+allow users to (in)validate their own DBS USER datasets
+"""
+
+import sys
+import json
+
+from CRABClient.Commands.SubCommand import SubCommand
+from CRABClient.ClientExceptions import MissingOptionException, ConfigurationException, CommandFailedException
+from CRABClient.ClientUtilities import colors
+from CRABClient.CrabRestInterface import HTTPRequests
+
+if sys.version_info >= (3, 0):
+    from urllib.parse import urlencode  # pylint: disable=E0611
+if sys.version_info < (3, 0):
+    from urllib import urlencode
+
+try:
+    from CRABClient import __version__
+except:  # pylint: disable=bare-except
+    __version__ = '0.0.0'
+
+
+def getDbsREST(instance=None, logger=None, cert=None, key=None, version=None):
+    """
+    given a DBS istance (e.g. prod/phys03) returns a DBSReader and DBSWriter
+    client instances which communicate with DBS REST via curl
+    Arguments:
+    logger: a logger
+    cert, key : name of files, can use the path to X509_USER_PROXY for both
+    version: the CRAB Client version to put in the User Agent field of the query
+    """
```

Comment:
Why do you need to explicitly set CRABClient version here instead of read from usual module version?

Reply:
VERY good question ! I suspect for no good reason, I will address toghether with "move to ClientUtilities"